modules/serverless-tls-config-additional-ca-bundles-eventing.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * /serverless/Eventing/serverless-config-tls-encryption-eventing.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="serverless-tls-config-additional-ca-bundles-eventing_{context}"]
+= Configuring additional CA trust bundles
+
+By default, Eventing clients trust the OpenShift CA bundle configured for custom PKI. For more details, see link:https://docs.openshift.com/container-platform/latest/networking/configuring-a-custom-pki.html[Configuring a custom PKI].
+
+[NOTE]
+====
+Eventing clients will automatically include these CA bundles in their trusted list when a new connection is established.
+====
+
+.Prerequisites
+
+* You have cluster administrator permissions on {ocp-product-title}, or you have cluster or dedicated administrator permissions on {rosa-product-title} or {dedicated-product-title}.
+* You have installed the {ServerlessOperatorName}.
+* You have installed the {cert-manager-operator}.
+
+.Procedure
+
+* Create a CA bundle for Eventing as follows:
++
+[source,yaml]
+----
+kind: ConfigMap
+metadata:
+  name: my-org-eventing-bundle <1>
+  namespace: knative-eventing
+  labels:
+    networking.knative.dev/trust-bundle: "true"
+data: <2>
+  ca.crt: ...
+  ca1.crt: ...
+  tls.crt: ...
+----
+<1> Use a unique name to avoid conflicts with existing or future Eventing `ConfigMaps`.
```

Comment:
Use the `ConfigMap` spelling only if you are referring to the actual `ConfigMap` object. For general references to config maps, use the spelling of "config map".